Tilers/CityTiler/CityTiler.py

```diff
@@ -154,7 +155,9 @@ def from_3dcitydb(cursor, objects_type):
 
         # Eventually we can add the newly build tile to the tile set:
         tileset.add_tile(tile)
-
+        if i == 15 :
```

Comment:
Is this debugging code ? And i a debugging index ?

Reply:
This code allowed us to chose how many tiles we wanted to create, in order to create a small tileset to see if our workflow worked. I believe it is still here because of two reasons : 
- It still allow us to create a small tileset, for debugging purpose
- We are currently having an error when trying to compute all of the tileset, must look into that in the near futur

Reply:
Fixed in 6a5720e9dc6ce7cbc139642aebd588fb47eb88ff

Reply:
Resolved in working session.

Reply:
Fixed together with CCO in working session.